src/Combinatorics/SimplicialComplexes.jl

```diff
@@ -621,3 +621,33 @@ function deletion(K::SimplicialComplex, face::Union{<:AbstractSet{Int},<:Abstrac
   zero_based = Polymake.to_zero_based_indexing(face)
   return SimplicialComplex(Polymake.topaz.deletion(pm_object(K), zero_based))
 end
+
+@doc raw"""
+    automorphism_group(K::SimplicialComplex)
```

Comment:
```suggestion
    automorphism_group(K::SimplicialComplex; action=:on_vertices)
```
And maybe mention/explain the action symbols in the docstring below?